src/App.tsx

```diff
@@ -42,23 +49,23 @@ export const App: React.FC = () => {
       <div className="buttons">
         <Button
           dataCy="all-button"
```

Comment:
```suggestion
          dataCy={ButtonType.ALL}
```
Can be added to all buttons